nautobot/dcim/templates/dcim/device.html

```diff
@@ -242,7 +242,7 @@
                             {% plugin_left_page object %}
                         </div>
                         <div class="col-md-6">
-                            {% if object.powerports.exists and object.poweroutlets.exists %}
+                            {% if object.power_ports.exists and object.power_outlets.exists %}
```

Comment:
Oof. Was this just a straight-up bug in all of 2.x? Let's make a note of it in the `.fixed` change fragment if so.